server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -205,11 +205,7 @@ export function configureWebSocketServices(
                 return Promise.reject(throttleError);
             }
             if (!message.token) {
-                // eslint-disable-next-line prefer-promise-reject-errors
-                return Promise.reject({
-                    code: 403,
-                    message: "Must provide an authorization token",
-                });
+                throw new NetworkError(403, "Must provide an authorization token");
```

Comment:
Follow-up (not in this PR):  Should this be 401?  I think of 401 as unauthenticated, and 403 as authenticated but unauthorized.  Missing token feels like 401.  This article has references to the most recent RFCs about these as well:  https://www.freecodecamp.org/news/http-401-error-vs-http-403-error-status-code-responses-explained/

Reply:
Also, looks like 401 and 403 are treated identically in the client code at the moment, so maybe we need to step back all up and think about what we want the end-to-end error flow to be in these cases.

Reply:
Actually, 401 is handled differently, it's just handled in a different place. In the driver, when a 401 is received, an [internal retry is performed after requesting a refreshed access token from TokenProvider](
https://github.com/microsoft/FluidFramework/blob/main/packages/drivers/routerlicious-driver/src/restWrapper.ts#L82-L86).

Reply:
I added a comment in `createR11sNetworkError` to make this knowledge more readily findable.